spec/etl/feedex/processor_spec.rb

```diff
@@ -23,25 +19,23 @@
     end
 
     it 'does not update metrics for other days' do
-      fact1 = create :metric, dimensions_item: item1, dimensions_date: dimensions_date, feedex_comments: 1
+      fact1 = create_metric base_path: '/path1', date: '2018-02-20', daily: { feedex_comments: 1 }
       day_before = date - 1
       described_class.process(date: day_before)
 
       expect(fact1.reload).to have_attributes(feedex_comments: 1)
     end
 
     it 'does not update metrics for other items' do
-      item = create :dimensions_item, base_path: '/non-matching-path', latest: true
-      fact = create :metric, dimensions_item: item, dimensions_date: dimensions_date, feedex_comments: 9
+      fact = create_metric base_path: '/non-matching-path', date: '2018-02-20', daily: { feedex_comments: 9 }
 
       described_class.process(date: date)
 
       expect(fact.reload).to have_attributes(feedex_comments: 9)
     end
 
     it 'deletes the events that matches the base_path of an item' do
-      item2.destroy
```

Comment:
Are we ok with removing the `item2.destroy` ? In theory it was there for a reason